### ISSUE TITLE
ECS broke honeypot in 1.1.1

### ISSUE THREAD

**Issue by andysh-uk** [bobdenotter/forms#39]
I've just updated my dev site to 1.1.1 to implement my CAPTCHA work in my own site, and noticed my honeypot field was always visible.

There is a Javascript error in the browser console stating it cannot find a particular field:

```
Uncaught TypeError: can't access property "style", el is null
```

This is in the Honeypot Javascript:

![image](https://user-images.githubusercontent.com/12907944/100883195-fafd0200-34a7-11eb-8a4b-bb1a5064e1cb.png)

The funny thing is this field doesn't actually exist - the Honeypot field is called something totally different.

I was looking through the commits in 1.1.1 to work out what had changed (as I didn't see this in my development of the CAPTCHA work) and found a change that ECS has made in [src/Honeypot.php](https://github.com/bolt/forms/blob/7d5099ae693e6b1da5dd7a35a731bdab52c51da6/src/Honeypot.php#L54:L56).

It looks like it has replaced `mt_rand` with `random_int` - luckily the comment explicitly states *why* you are using `mt_rand`!!

Reverting this change fixes my Honeypot field.

**Comment by bobdenotter** [comment]
Yes, let's revert that part!

**Comment by andysh-uk** [comment]
Thanks guys, I can confirm this is resolved in 1.1.2.

**Comment by I-Valchev** [comment]
Nice! Glad to hear @andysh-uk
